tests/acceptance/features/apiAntivirus/antivirusMain.feature

```diff
@@ -121,7 +121,6 @@ Feature: Antivirus basic
       | eicar_com.zip   |
       | eicarcom2.zip   |
 
-  @skip @issue-334
   Scenario Outline: A small file with a virus cannot be uploaded via new public upload
     Given the administrator has enabled DAV tech_preview
```

Comment:
@phil-davis I think I should delete this line too as tech preview parameter  is not needed anymore.

Reply:
@VicDeo yes, any reference to tech_preview can be removed.